### ISSUE TITLE
Anyone tried this model to write a next sentence?

### ISSUE THREAD

**Issue by qnkhuat** [huggingface/pytorch-pretrained-BERT#285]

**Comment by thomwolf** [comment]
Closing for now.

**Comment by qnkhuat** [comment]
Why ? @thomwolf

**Comment by thomwolf** [comment]
I'm trying to clean up the issues to get a better view of what needs to be fixed.
But you are right opening/closing issue is too binary. Let's add labels instead.
